doc/explanations/faq.rst

```diff
@@ -1,218 +1,337 @@
+policy.
 
-What do Plutus smart contracts have to do with Native Tokens?
-********************************************************************************
+**Q. What do Plutus smart contracts have to do with native tokens?**
 
-Minting policies can be written in the Plutus smart contract language. This allows users to express a much wider range of policies than just the single issuer policy expressible using multisig. The one-time minting policy, for example, can be expressed in Plutus (but not just as multisig).
+A. Minting policies can be written in the Plutus smart contract
+language. This allows users to express a much wider range of policies
+than just the single issuer policy expressible using multisig. The
+one-time minting policy, for example, can be expressed in Plutus (but
+not just as multisig).
 
-What is a single-issuer minting policy?
-********************************************************************************
+**Q. What is a single-issuer minting policy?**
 
-A single-issuer minting policy specifies that only the entity holding a particular set of keys is allowed to mint tokens under a particular policy. For example, the set of keys specified in the minting policy must have signed the minting transaction. This type of policy can be specified using multisig.
+A. A single-issuer minting policy specifies that only the entity holding
+a particular set of keys is allowed to mint tokens under a particular
+policy. For example, the set of keys specified in the minting policy
+must have signed the minting transaction. This type of policy can be
+specified using multisig.
 
-An example of a single-issuer policy use case could be tokens representing baseball cards. This would mean that no new baseball card tokens could be minted without the company’s signatures. Conversely, the policy proves that all the existing cards scoped under this policy have been legitimately minted by the baseball card company.
+An example of a single-issuer policy use case could be tokens
+representing baseball cards. This would mean that no new baseball card
+tokens could be minted without the company’s signatures. Conversely, the
+policy proves that all the existing cards scoped under this policy have
+been legitimately minted by the baseball card company.
 
-What is a one-time minting policy?
-********************************************************************************
+**Q. What is a one-time minting policy?**
 
-In a one-time minting policy, the complete set of tokens scoped under it is minted by one specific transaction. This means that no more tokens will ever be minted under that policy. This type of policy does require smart contracts and cannot be expressed using multisig.
+A. In a one-time minting policy, the complete set of tokens scoped under
+it is minted by one specific transaction. This means that no more tokens
+will ever be minted under that policy. This type of policy does require
+smart contracts and cannot be expressed using multisig.
```

Comment:
The multi-sig DSL was expanded in the Allega ledger era to include timelocks, which do provide the ability to guarantee that a given policy can never again be used after a gives slot.